proto/info/info.proto

```diff
@@ -214,6 +214,9 @@ enum ZSwState {
   // START_DELAYED state after ERROR.
   // Had to prefix with Z_SW_STATE_ to satisfy yetus checks.
   START_DELAYED = 22; // App has been configured to start with delay by user
+
+  Z_SW_STATE_PENDING = 23; // Kubernetes app in pending state
+  Z_SW_STATE_SCHEDULING = 24; // Kubernetes app in scheduling state
```

Comment:
It's better to ignore the yetus complaint about the prefix to make all of the values in a given enum follow the same style. So PENDING and SCHEDULING makes more sense as names.

Reply:
OK I thought you told me to add the prefix for new enums. I will change that and resubmit. BTW, I did publish the new make proto files too. Not sure why it is complaining.